File: 04.py
decimal = int(input("Introduce un número decimal: "))


def convertir_a_binario(decimal):
    binario = ""
    while decimal // 2 != 0:
        binario += str(decimal % 2)
        decimal = decimal // 2

    print((binario + str(decimal))[::-1])


convertir_a_binario(decimal)

File: 04.js
let decimal = Number(prompt('Introduce un número decimal: '));

function binario(decimal) {
  let resultado = [];
  while (decimal > 0) {
    resultado.unshift(decimal % 2);
    decimal = Math.floor(decimal / 2);
  }
  return resultado.join('');
}

console.log(binario(decimal));
